services/server/server.go

```diff
@@ -508,6 +528,29 @@ func (s *Server) ChangeSettings(ctx context.Context, req *serverpb.ChangeSetting
 			settings.SSHKey = req.SshKey
 		}
 
+		if req.AlertManagerRules != "" && !req.RemoveAlertManagerRules {
+			if e := validateRulesFile(req.AlertManagerRules); e != nil {
+				return e
+			}
+			if e := ioutil.WriteFile(s.alertsManagerFile, []byte(req.AlertManagerRules), os.ModePerm); e != nil {
+				return e
+			}
+			if e := os.Chmod(s.alertsManagerFile, 0644); e != nil {
+				s.l.Warn(e)
+			}
+		}
+
+		if req.RemoveAlertManagerRules {
+			if e := os.Remove(s.alertsManagerFile); e != nil {
+				s.l.Warn(e)
+			}
+		}
+
+		settings.AlertManagerAddress = req.AlertManagerAddress
```

Comment:
We can send settings separately, so `req.AlertManagerAddress` can be empty and it will remove `settings.AlertManagerAddress`. Let's add check that `req.AlertManagerAddress` isn't empty.